src/makerspace/templates/makerspace/statistics.html

```diff
@@ -0,0 +1,68 @@
+{% extends 'web/base.html' %}
+{% load static %}
+{% load i18n %}
+
+
+{% block title %}{% translate "Statistics" %}{% endblock title %}
+
+{% block extra_head %}
+    <script defer src="{% static 'makerspace/js/statistics_charts.js' %}"></script>
+{% endblock extra_head %}
+
+{% block body%}
+<div class="ui container">
+    <h1 class="ui text-centered make-col-blue header"> {% translate "Statistics" %}</h1>
+    <div class="ui column cards">
+    <div class="card " style="height: 50vh; width: 65vh">
```

Comment:
Generally on the MAKE website, styles are put in separate CSS documents. Maybe that should be done here as well, instead of styling each element with the same CSS code.